packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/description/DiagramDescription.java

```diff
@@ -142,6 +149,8 @@ public static final class Builder {
 
         private boolean autoLayout;
 
+        private ArrangeLayoutDirection arrangeLayoutDirection;
```

Comment:
Related to my previous comment in the changelog, the default value would be here